docs/guide/use_cases/nlb_tls_termination.md

```diff
@@ -0,0 +1,129 @@
+
+If the domain is registered with Route53, the TLS certificate request will automatically be approved. Otherwise, follow ACM console the instructions to create a DNS record to validate the domain. 
+
+After validation, the certificate will be available for use in your AWS account. 
+
+Note the ARN of the certificate, which uniquely identifies it in kubernetes config files. 
+
+![screenshot indicating location of ARN value in web console](cert_arn.png)
+
+### Create Service with new NLB
+
+Add annotations to a load balancer service to enable NLB TLS termination, before the traffic reaches Envoy. The annotations are actioned by the load balancer controller. [Review all the NLB annotations on GitHub.](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/service/annotations/)
+
+| annotation name | value | meaning | 
+| ----- | --- | ----- |
+| service.beta.kubernetes.io/aws-load-balancer-type | external | explicitly requires an NLB, instead of an ALB |
+| service.beta.kubernetes.io/aws-load-balancer-nlb-target-type | ip | route traffic directly to the pod IP |
+| service.beta.kubernetes.io/aws-load-balancer-scheme | internet-facing | An internet-facing load balancer has a publicly resolvable DNS name |
+| service.beta.kubernetes.io/aws-load-balancer-ssl-cert | "arn:aws:acm:..." | identifies the TLS certificate used by the NLB |
+| service.beta.kubernetes.io/aws-load-balancer-ssl-ports | 443 | determines the port the NLB should listen for TLS traffic on| 
+
+Example: 
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: MyAppSvc
+  namespace: dev
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-type: external
+    service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: ip
+    service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "arn:aws:acm:us-east-2:185309785115:certificate/7610ed7d-5a81-4ea2-a18a-7ba1606cca3e"
+    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
+spec:
+  externalTrafficPolicy: Local
+  ports:
+  - port: 443
+    targetPort: 80
+    name: http
+    protocol: TCP
+  selector:
+    app: MyApp
+  type: LoadBalancer
+```
+
+!!! warning
+    Don't modify an existing service to add NLB TLS termination. This may result in unexpected behavior, such as duplicate NLB resources or incorrect NLB configuration. 
+
+### Configure DNS
+
+**Get domain name using kubectl.** 
+
+The service name and namespace were defined above.
+
+```
+kubectl get svc MyAppSvc --namespace dev
+```
+
+```
+NAME    TYPE           CLUSTER-IP      EXTERNAL-IP                                                                     PORT(S)        AGE
+envoy   LoadBalancer   10.100.24.154   a7ea2bbde8a164036a7e4c1ed5700cdf-154fb911d990bb1f.elb.us-east-2.amazonaws.com   443:31606/TCP   40d
```

Comment:
By default, an NLB provisioned by this controller has the name format `k8s-<namespace>-<name>-<hash>`. Looking at the name it appears to be provisioned by kcm. Lets provide an example from a service provisioned by this controller to avoid confusion.

Reply:
excellent point thank you